pyterrier/pipelines.py

```diff
@@ -45,21 +50,90 @@ def _color_cols(data, col_type,
     is_max[len(data) - list(reversed(data)).index(max_value) -  1] = colormaxlast_attr
     return is_max
 
+_measure_nicknames = {
```

Comment:
We can drop `_measure_nicknames` and `_measure_prefix` now, right?

Reply:
Good spot. it seems likely!